frontend/components/Layout/NavBar.tsx

```diff
@@ -52,6 +74,8 @@ export const NavBar = () => {
         <YellowLight onClick={() => electronApi?.minimizeApp?.()} />
         <DisabledLight />
       </TrafficLights>
+
+      <PearHeader />
```

Comment:
can we migrate all the changes done for on-ramping here? and also somehow consider showing this only for onboarding pages (or not showing at all, I talked to Roman re this and he's fine if the logo is not present)